### PR TITLE
Choose preferred worker from worker list

### PR DESCRIPTION
**Pull request by huanghua78**
### What changes are proposed in this pull request?

Chooses a preferred worker from worker list to fetch files

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -5165,7 +5165,7 @@ public String toString() {
           .build();
   public static final PropertyKey USER_FILE_REPLICATION_MIN =
       intBuilder(Name.USER_FILE_REPLICATION_MIN)
-          .setDefaultValue(0)
+          .setDefaultValue(1)
```
Comment (dbw9580) [inline dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:5168]: the default value should stay `0` as a minimum of 1 replica means the file always has to be cached on at least one worker. This prevents the pages of the file from being evicted.

**Review by dbw9580 — APPROVED** [review]
LGTM except for the commnet above. Thanks!

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (huanghua78) [reply dora/core/common/src/main/java/alluxio/conf/PropertyKey.java]: OK

Comment (huanghua78) [reply dora/core/common/src/main/java/alluxio/conf/PropertyKey.java]: @dbw9580 Please take another look.

File: dora/core/client/fs/src/main/java/alluxio/client/file/dora/DoraCacheClient.java [hunk 2]
```diff
@@ -388,14 +391,33 @@ public WorkerNetAddress getWorkerNetAddress(String path) {
...
+  }
+
+  /**
+   * Get Context.
+   *
+   * @return a file system context
+   */
+  public FileSystemContext getContext() {
+    return mContext;
+  }
```
Comment (dbw9580) [inline dora/core/client/fs/src/main/java/alluxio/client/file/dora/DoraCacheClient.java:422]: normally the one who created `DoraCacheClient` should have a reference to `FileSystemContext`, so you don't need to expose it via a public getter method. How is this method necessary?

Comment (huanghua78) [reply dora/core/client/fs/src/main/java/alluxio/client/file/dora/DoraCacheClient.java:422]: This class will be extended in other places. The extended class needs to access the context and then access the mID of the context.

**Comment by huanghua78** [comment]
alluxio-bot, merge this please

**Pull request merged**